### PR TITLE
Disable Android framework detection for Flutter projects

### PR DESCRIPTION
**Pull request by stevemessick**

### CODE REVIEW

File: src/io/flutter/ProjectOpenActivity.java [hunk 1]
```diff
@@ -101,10 +101,14 @@ public void runActivity(@NotNull Project project) {
...
-    if (!excludesConfiguration.isExcludedFromDetection(type)) {
-      excludesConfiguration.addExcludedFramework(type);
+    try {
+      final DetectionExcludesConfiguration excludesConfiguration = DetectionExcludesConfiguration.getInstance(project);
+      final FrameworkType type = new AndroidFrameworkDetector().getFrameworkType();
+      if (!excludesConfiguration.isExcludedFromDetection(type)) {
+        excludesConfiguration.addExcludedFramework(type);
+      }
+    } catch (NoClassDefFoundError ignored) {
+      // This should never happen. But just in case ...
```
Comment (jacob314) [inline src/io/flutter/ProjectOpenActivity.java:111]: nit: mention the bug where this has happened.

**Review by jacob314 — APPROVED** [review]
lgtm

**Comment by stevemessick** [comment]
Thanks! This fixed half the problems reported in #5132

**Comment by stevemessick** [comment]
This change should be part of the next dev channel update this weekend.

**Pull request merged**